resolwe/flow/models/annotations.py

```diff
@@ -454,6 +454,15 @@ class Meta:
         ]
         ordering = ["field__group__sort_order", "field__sort_order"]
 
+    #: URL slug
+    slug = ResolweSlugField(
+        populate_from=lambda value: (
+            f"{value.entity.slug}-{value.field.group.name}" f"-{value.field.name}"
```

Comment:
Can you join this into a single sting?

Reply:
This is different now due to migration problems, they do not like lambdas.